docs/30_getting-started-guide/index.md

```diff
@@ -3,12 +3,27 @@ content_title: "Getting Started Guide"
 link_text: "Getting Started Guide"
 ---
 
-The EOSIO _Getting Started Guide_ provides guided instructions on how to set up your local development environment. These instructions installs the EOSIO software components and other related dependencies for the EOSIO platform. This setup is an efficient and convenient method to onboard the EOSIO development ecosystem. Subsequently, you become acquainted with the smart contracts development workflow in a local environment.
+The purpose of the _EOSIO Getting Started Guide_ is to provide guided instructions to set up your local EOSIO development environment and use the environment to build and deploy a basic **_Hello World_** Smart Contract. The _Getting Started Guide_ high-level workflow is illustrated in the following diagram: 
+
+![EOSIO Getting Started Workflow](../eosio-gsg-workflow.png)
+
+The diagram identifies the following two workflows: 
+
+1. **Set up local development environment**
+2. **Build and deploy a basic Hello World smart contract**
+
+[[info | Note]]
+| Before you begin with the getting started workflow, make sure you familiarize yourself with [EOSIO Platform and Toolchain](../20_introduction-to-eosio/10_platform_and_toolchain.md), [Core Concepts](../20_introduction-to-eosio/20_core_concepts.md), and some of the [Technical Features](20_introduction-to-eosio/30_technical_features.md).
 
 # Set Up Local Development Environment
-As a developer, begin with setting up your [local development environment](20_local-development-environment) that involves satisfying system requirements, installing OS-specific EOSIO binaries, and creating test blockchain accounts. Once the local development environment is set up, you can progress to smart contracts development workflow. 
+As a developer, begin with setting up your [local development environment](../30_getting-started-guide/20_local-development-environment/index.md) that involves satisfying system requirements, installing OS-specific EOSIO binaries, and creating test blockchain accounts. Once the local development environment is set up, you can progress to smart contracts development workflow and build your first basic **Hello World** smart contract.
```

Comment:
Sentence doesn't read right ... how about ->
As a developer, begin with setting up your [local development environment.](../30_getting-started-guide/20_local-development-environment/index.md) To do this satisfy the system requirements, install the OS-specific EOSIO binaries, and create test blockchain accounts.

Reply:
I think both the versions sounds fine. The existing and your suggestion also.